source/reference/operator/update/addToSet.txt

```diff
@@ -33,12 +33,36 @@ elements in the modified set.
 If the field is absent in the document to update, :update:`$addToSet`
 creates the array field with the specified value as its element.
 
-If the field is **not** an array, the operation will fail.
+If the field is **not** an array, the operation will fail with an
+error message stating that :update:`$addToSet` cannot be applied to a
+non-array field.
 
 If the value is an array, :update:`$addToSet` appends the whole array
-as a *single* element. To add each element of the value separately, use
-the :update:`$each` modifier with :update:`$addToSet`. See
-:ref:`addToSet-modifiers` for details.
+as a *single* element. For instance, consider a document in a
+collection ``test`` containing an array field ``letters``:
+
+.. code-block:: javascript
+
+   {"_id": 1, "letters": ["a", "b"]}
+
+Calling
+
+.. code-block:: javascript
+
+ db.test.update(
+   {"_id": 1 },
+   {$addToSet: {"letters": ["c" , "d"]}}
+ )
+
+results in
```

Comment:
these fragment phrases without colons or capitalization are not consistent with our style, and we should tighten up these examples.